waves-ext/src/main/scala/com/wavesplatform/dex/grpc/integration/DEXExtension.scala

```diff
@@ -31,8 +32,12 @@ class DEXExtension(context: ExtensionContext) extends Extension with ScorexLoggi
     val host: String = context.settings.config.as[String]("waves.dex.grpc.integration.host")
     val port: Int    = context.settings.config.as[Int]("waves.dex.grpc.integration.port")
 
+    val ignoredExchangeTxSenderPublicKey: Option[String] = Try {
+      context.settings.config.as[String]("waves.utx.ignore-exchange-sender-pk-in-pessimistic-portfolio")
```

Comment:
This setting should be a part of `waves.dex` at least.

Reply:
Done